.github/CONTRIBUTING.md

```diff
@@ -18,13 +18,16 @@ Please fill in the description following the template.
 
 4.  Create a change log entry for the release:
 
-    - You'll need an [access token for the GitHub API](https://help.github.com/articles/creating-an-access-token-for-command-line-use/). Save it to this environment variable: `export GITHUB_AUTH="..."`
+    - You'll need an [access token for the GitHub API](https://help.github.com/articles/creating-an-access-token-for-command-line-use/). This token should have all the "repo" permission granted (no need for other permissions). Save it to this environment variable: `export GITHUB_AUTH="..."`
```

Comment:
I was not sure about the scope/permissions part here, so I checked what are the minimum requirements, which is best to have in terms of security.
Turns out that all repo are needed for private repos, whereas only 1 will suffice for public repos, such as the ECL one.
I think it'd be best for the token explanation note to refer to this https://github.com/lerna/lerna-changelog#github-token This way we are safe in time regardless whether the repo remain public or not, etc.